common/workunit/workunit.cpp

```diff
@@ -4049,6 +4095,20 @@ class CLockedWorkUnit : implements ILocalWorkUnit, implements IExtendedWUInterfa
             { return c->getAbortBy(str); }
     virtual unsigned __int64 getAbortTimeStamp() const
             { return c->getAbortTimeStamp(); }
+    virtual StringBuffer & getProcessLogPattern(const char * type, const char *instance, StringBuffer & pattern) const
+            { return c->getProcessLogPattern(type, instance, pattern); }
+    virtual void setProcessLogPattern(const char * type, const char *instance, const char * pattern)
+            { return c->setProcessLogPattern(type, instance, pattern); }
```

Comment:
it looks like neither of these are actually used?
I think the log @pattern - is (and should be) only for internal use, so there shouldn't be exposed interface methods.

[not sure if that also applies to some of the other methods below]